Gemfile.jruby-1.9.lock.release

```diff
@@ -185,6 +185,11 @@ GEM
       rspec (~> 3.0)
       rspec-wait
       stud (>= 0.0.20)
+    logstash-filter-aggregate (2.7.2)
```

Comment:
It is odd to see us adding so many more default plugins here.  How did you generate this lock file ?